### PR TITLE
Append /opt/conda/bin to the sudo secure_path

### LINKED ISSUE #524: sudo: jupyter: command not found
Hi! Thanks for using Jupyter's docker-stacks images.

If you are requesting a library upgrade or addition in one of the existing images, please state the desired library name and version here and disregard the remaining sections.

If you are reporting an issue with one of the existing images, please answer the questions below to help us troubleshoot the problem. Please be as thorough as possible.

**What docker image you are using?**

`jupyter/datascience-notebook`

**What complete docker command do you run to launch the container (omitting sensitive values)?**

`docker run -dit --name datascience --user root -e GRANT_SUDO=yes -e NB_UID=1000 -e NB_GID=1000 --restart always -p 80:8888 -v /vagrant:/home/jovyan/work start-notebook.sh --NotebookApp.notebook_dir=work --NotebookApp.token=''`

**What steps do you take once the container is running to reproduce the issue?**

docker logs datascience

Set username to: jovyan
usermod: no changes
Set jovyan GID to: 1000
Granting jovyan sudo access
Executing the command: jupyter notebook --NotebookApp.notebook_dir=work --NotebookApp.token=
sudo: jupyter: command not found
Set username to: jovyan
usermod: no changes
Granting jovyan sudo access

**What do you expect to happen?**

see the jupyter notebook initialization logs

**What actually happens?**

the server does not start as the sudo user does not find jupyter

### PR DESCRIPTION
**Pull request by parente**
Fixes #524. Adds a test to prevent regression.

### CODE REVIEW

File: base-notebook/test/test_container_options.py [hunk 1]
```diff
@@ -71,6 +71,19 @@ def test_sudo(container):
...
     assert 'uid=0(root)' in c.logs(stdout=True).decode('utf-8')
 
+
+def test_sudo_path(container):
+    """Container should include /opt/conda/bin in the sudo secure_path."""
+    c = container.run(
+        tty=True,
+        user='root',
+        environment=['GRANT_SUDO=yes'],
+        command=['start.sh', 'sudo', 'jupyter', '--version']
```
Comment (ericdill) [inline base-notebook/test/test_container_options.py]: would it be possible to capture the output of ``which jupyter`` and make sure that it starts with ``/opt/conda/bin``? Does that have any advantages?

Comment (ericdill) [inline base-notebook/test/test_container_options.py]: would it be possible to capture the output of ``which jupyter`` and make sure that it starts with ``/opt/conda/bin``? Does that have any advantages?

**Comment by lhoss** [comment]
@parente +1 on the quick fix+merge 
except it seems only the `base-notebook` build worked and got published on dockerhub, but not the others (for ex the pyspark one, I'm using, latest tag 6d old: https://hub.docker.com/r/jupyter/pyspark-notebook/tags/ )

**Comment by parente** [comment]
@lhoss Thanks for the check-up. I'll retry the builds this morning. Docker Cloud sometimes fails with  out of disk space or other infrastructure errors.

**Comment by parente** [comment]
@ericdill Thanks for the look-see. 🍰

**Comment by parente** [comment]
jupyter/minimal notebook failed and held up the rest of the builds:

https://cloud.docker.com/app/jupyter/repository/registry-1.docker.io/jupyter/minimal-notebook/builds/ad23be1d-cfcd-41ab-b8d5-686960517a7b

Retrying now.

**Pull request merged**